example/openapi.yml

```diff
@@ -1,4 +1,4 @@
-openapi: 3.0.3
+openapi: 3.1.0
```

Comment:
Just to have stronger tests, can we store both versions and run tests with both versions?

Reply:
[openapi-schema-to-json-schema](https://github.com/openapi-contrib/openapi-schema-to-json-schema) supports OpenAPI 3.0, not OpenAPI 3.1. 

Until `openapi-schema-to-json-schema` supports OpenAPI 3.1 and newer versions of JSON Schema, I don't think suggesting this tool supports OpenAPI 3.1 is safe. Maybe someone knows of a better option that supports OpenAPI 3.0 (required for some API gateways) and OpenAPI 3.1 and can add an issue to switch to it.

Among key differences:

- 3.1 supports the `examples` (multiple examples) key in schemas; 3.0 supports `example` (single example) only. I know the difference causes issues when building proxies in some API gateways. I think `openapi-schema-to-json-schema` handles it, but can't remember.
- 3.0 supports `nullable: true`; 3.1 does not support `nullable`, preferring a compound (array or `oneOf` type, see link in notes added to README); `openapi-schema-to-json-schema` fails with the array type option even though it produces the same output as `nullable: true`.

Reply:
Since it's not safe, why use it for the tests and examples? It's misleading.